apps/calendar/locales/calendar.en-US.properties

```diff
@@ -32,7 +32,9 @@ account-header=Account
 account-list-header=Accounts
 
 # advanced settings
-advanced-settings-short=Settings
+# This settings button uses an icon for display, so this text is only
+# for accessibility purposes, not visible to the eye.
```

Comment:
Use the default comment for screen reader strings: https://github.com/jobara/gaia/blob/1102506/apps/calendar/locales/calendar.en-US.properties#L85-L86